frontend/pages/schedule/ManageSchedulePage/ManageSchedulePage.tsx

```diff
@@ -167,31 +173,39 @@ const ManageSchedulePage = ({
     }
   );
 
-  const teamId = team_id ? parseInt(team_id, 10) : 0;
+  let selectedTeamId: number;
 
-  const handleTeamSelect = (selectedTeamId: number) => {
-    if (selectedTeamId) {
-      dispatch(push(MANAGE_TEAM_SCHEDULE(selectedTeamId)));
+  if (currentTeam) {
+    selectedTeamId = currentTeam.id;
+  } else {
+    selectedTeamId = team_id ? parseInt(team_id, 10) : 0;
+  }
+
+  const handleTeamSelect = (teamId: number) => {
+    if (teamId) {
+      dispatch(push(MANAGE_TEAM_SCHEDULE(teamId)));
```

Comment:
I know we're trying to replace these old dispatch/redux patterns with router.push

Reply:
Thanks for the reminder, I'll keep that in mind as I update.